### PR TITLE
[DOC] Update ReadMe & add Contributing guide

### PR DESCRIPTION
**Pull request by csouchet**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+[Fork bpmn-visualization-r](https://help.github.com/articles/fork-a-repo) and create a branch with a descriptive name. 
+
+A good branch name would be (where issue #25 is the ticket you're working on): **25-customize_overlays**
+
+```sh
+git checkout -b 25-customize_overlays
+```
+
+### IDE configuration
+To know how to configure your development environment, see [IDE configuration](https://github.com/process-analytics/bpmn-visualization-js/blob/master/docs/development/ide-configuration.md)
```
Comment (tbouffard) [inline CONTRIBUTING.md:46]: ⚠️ not releavant here: the link is for JS/TS development

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+    <p align="center">
+        <a href="https://github.com/process-analytics/bpmn-visualization-r/releases">
+          <img alt="GitHub release (latest by date including pre-releases)" src="https://img.shields.io/github/v/release/process-analytics/bpmn-visualization-r?label=changelog&include_prereleases"> 
+        </a>
+        <a href="https://github.com/process-analytics/bpmn-visualization-r/actions">
+          <img alt="Build" src="https://github.com/process-analytics/bpmn-visualization-r/workflows/Build/badge.svg"> 
+        </a> 
+        <a href="https://gitpod.io/#https://github.com/process-analytics/bpmn-visualization-r" target="_blank">
+          <img alt="Gitpod" src="https://img.shields.io/badge/Gitpod-ready--to--code-chartreuse?logo=gitpod"> 
+        </a>
```
Comment (tbouffard) [inline README.md:12]: ❓ not sure gitpod is able to manage R environment, especially without specific extensions installation

File: README.md [hunk 3]
```diff
@@ -1,2 +1,27 @@
...
+          <img alt="Contributor Covenant" src="https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg"> 
+        </a> 
+        <a href="LICENSE">
+          <img alt="License" src="https://img.shields.io/github/license/process-analytics/bpmn-visualization-r?color=blue"> 
+        </a>
+    </p>
+</div>  
+<br>
+
 A R package which embeds BPMN Visualization
```
Comment (tbouffard) [inline README.md:27]: To be improved in the future: bpmn-visualization is the way used by the package to display execution data over BPMN Diagrams

File: README.md [hunk 4]
```diff
@@ -1,2 +1,27 @@
-# bpmn-visualization-R
+<h1 align="center">BPMN Visualization - R Package</h1>
+<div align="center">
+    <p align="center">
+        <a href="https://github.com/process-analytics/bpmn-visualization-r/releases">
+          <img alt="GitHub release (latest by date including pre-releases)" src="https://img.shields.io/github/v/release/process-analytics/bpmn-visualization-r?label=changelog&include_prereleases"> 
+        </a>
+        <a href="https://github.com/process-analytics/bpmn-visualization-r/actions">
+          <img alt="Build" src="https://github.com/process-analytics/bpmn-visualization-r/workflows/Build/badge.svg"> 
+        </a>
```
Comment (tbouffard) [inline README.md:9]: ⚠️ this icon will be broken until we have a GH WF configured in this repo

**Review by tbouffard — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (csouchet) [reply README.md:9]: Yes, I though to configure the worklow later ^^

**Review by tbouffard — APPROVED** [review]

**Pull request merged**